fontir/src/stateset.rs

```diff
@@ -251,7 +251,7 @@ mod tests {
     use filetime::set_file_mtime;
     use tempfile::{tempdir, TempDir};
 
-    use super::{StateDiff, StateIdentifier, StateSet};
+    use super::*;
```

Comment:
nit: not a fan of this

Reply:
This is a standard convention for the test module, you'll see it in lots of rust projects.

The basic idea is: assume the test module has access to everything in the parent.

Reply:
I get the idea but I still don't like it. Where is this standard documented? - it seems suspiciously like something that is largely up to project owners preference.